app/services/config-persistence/nodes/sources.ts

```diff
@@ -142,7 +142,7 @@ export class SourcesNode extends Node<ISchema, {}> {
         }
       );
       if (source.audioMixers) {
-        this.audioService.getSource(sourceInfo.id).setMul(sourceInfo.volume);
+        this.audioService.getSource(sourceInfo.id).setMul(sourceInfo.volume || 1);
```

Comment:
I think this is a destructive change.  If people have their volume set to `0`, it will revert back to full volume.